src/components/FilterDropdown.jsx

```diff
@@ -45,10 +45,11 @@ FilterDropdown.propTypes = {
   label: PropTypes.string.isRequired,
   defaultSelect: PropTypes.string,
   items: PropTypes.objectOf(PropTypes.string).isRequired,
-  removeNone: PropTypes.bool.isRequired,
```

Comment:
This was marked required but not actually passed in all `FilterDropdown` places. Probably shouldn't be required.

Reply:
Yeah its no longer required with my PR being merged to main it will fix this since it failed a test when I attempted to push today